org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/WorkspaceManager.xtend

```diff
@@ -85,7 +86,7 @@ import org.eclipse.xtext.ide.server.BuildManager.Buildable
     protected def refreshWorkspaceConfig(CancelIndicator cancelIndicator) {
         workspaceConfig = workspaceConfigFactory.getWorkspaceConfig(baseDir)
         val newProjects = newArrayList
-        val remainingProjectNames = newHashSet(projectName2ProjectManager.keySet)
+        val remainingProjectNames = new HashSet(projectName2ProjectManager.keySet)
```

Comment:
Please make the type of the local var explicit so that subsequent changes do not revert the fix by accident.